dask_cuda/cli/dask_cuda_worker.py

```diff
@@ -19,194 +19,202 @@
+
+    .. warning::
+        Managed memory is currently incompatible with NVLink. Trying to enable both will
+        result in an exception.""",
 )
 @click.option(
     "--rmm-log-directory",
     default=None,
-    help="Directory to write per-worker RMM log files to; the client "
-    "and scheduler are not logged here."
-    "NOTE: Logging will only be enabled if --rmm-pool-size or "
-    "--rmm-managed-memory are specified.",
+    help="""Directory to write per-worker RMM log files to. The client and scheduler are
+    not logged here.
+
+    .. note::
+        Logging will only be enabled if ``--rmm-pool-size`` or ``--rmm-managed-memory``
+        are specified.""",
 )
+@click.option("--pid-file", type=str, default="", help="File to write the process PID.")
 @click.option(
-    "--reconnect/--no-reconnect",
-    default=True,
-    help="Reconnect to scheduler if disconnected",
+    "--resources",
+    type=str,
+    default="",
+    help="""Resources for task constraints like ``GPU=2 MEM=10e9``. Resources are
+    applied separately to each worker process (only relevant when starting multiple
+    worker processes with ``--nprocs``).""",
 )
-@click.option("--pid-file", type=str, default="", help="File to write the process PID")
 @click.option(
-    "--local-directory", default=None, type=str, help="Directory to place worker files"
+    "--dashboard/--no-dashboard",
+    "dashboard",
+    default=True,
+    show_default=True,
+    required=False,
+    help="Launch the dashboard.",
 )
 @click.option(
-    "--resources",
+    "--dashboard-address",
     type=str,
-    default="",
-    help='Resources for task constraints like "GPU=2 MEM=10e9". '
-    "Resources are applied separately to each worker process "
-    "(only relevant when starting multiple worker processes with '--nprocs').",
+    default=":0",
+    show_default=True,
+    help="Relative address to serve the dashboard (if enabled).",
+)
+@click.option(
+    "--local-directory", default=None, type=str, help="Directory to place worker files."
 )
 @click.option(
     "--scheduler-file",
     type=str,
     default="",
-    help="Filename to JSON encoded scheduler information. "
-    "Use with dask-scheduler --scheduler-file",
+    help="""Filename to JSON encoded scheduler information. To be used in conjunction
+    with the equivalent ``dask-scheduler`` option.""",
 )
 @click.option(
-    "--death-timeout",
+    "--interface",
     type=str,
     default=None,
-    help="Seconds to wait for a scheduler before closing",
+    help="""External interface used to connect to the scheduler. Usually an ethernet
+    interface is used for connection, and not an InfiniBand interface (if one is
+    available).""",
 )
 @click.option(
-    "--dashboard-prefix", type=str, default=None, help="Prefix for the Dashboard"
+    "--death-timeout",
+    type=str,
+    default=None,
+    help="Seconds to wait for a scheduler before closing.",
```

Comment:
It looks like this argument isn't used in `CUDAWorker` - I think we would want to pass this on to the worker's nanny, since that's where I believe the timeout is implemented.

Reply:
If it's not even being passed and nobody complained for so long it probably means it's not important, let's drop it instead.

Reply:
It probably would've been easier to make a list of all the kwargs to drop before opening a PR 😅 I'll address this in a separate PR since #562 is nearly ready to merge.

Reply:
Sounds good to me.